src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Listeners/EventHubListener.cs

```diff
@@ -336,6 +339,12 @@ private bool TryGetLinkedActivity(EventData message, out Activity link)
                 {
                     link = new Activity("Microsoft.Azure.EventHubs.Process");
                     link.SetParentId(diagnosticIdString);
+                    if (message.SystemProperties != null)
+                    {
+                        link.AddTag("EnqueuedTime", ((DateTimeOffset) message.SystemProperties.EnqueuedTimeUtc).ToUnixTimeMilliseconds()

```

Comment:
why do we use UnixTimeMilliseconds here? usually we log time as date-time string.